uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/AbsenceType.tpl

```diff
@@ -16,7 +16,6 @@
             <thead class="sticky">
               <th>{ts}Title{/ts}</th>
               <th>{ts}Allow Accruals?{/ts}</th>
-              <th>{ts}Is default?{/ts}</th>
```

Comment:
Does this mean that we would no longer be needing this field ever again (please confirm)? 
In that case it may be a good idea to remove the field from the db and also from the DAO and clean up code that still use the field.

Reply:
Yes we will not. It was said that 

> is_default is problematic as actually different users may have different sets of leave, and may have more than one leave type in a category

I will add clean up update.